doctrine.rst

```diff
@@ -654,6 +654,8 @@ In addition to the query builder, you can also query with `Doctrine Query Langua
 
     public function findAllGreaterThanPrice($price): array
     {
+        $em = $this->getEntityManager();
+        
         $query = $em->createQuery(
```

Comment:
I think we could just use `$this->_em` here instead.

Reply:
I'm not sure if this is a good idea, since $this->_em has never been introduced before. Without you mentioning it I wouldn't know it existed. The other code example in the same section doesn't use it as well.

Reply:
That's true! And in the example below, we use `$conn = $this->getEntityManager()->getConnection();`

Reply:
Let's merge this "as is" ... and later we can think of explaining `$this->_em` somewhere and using it in all examples.